protocol/src/common/protocol.resolveMoniker.proposed.md

```diff
@@ -0,0 +1,28 @@
+#### Resolve Symbol Moniker (textDocument/resolveMoniker)
```

Comment:
Opt to name this textDocument/moniker. In LSP we use the name resolve to compute additional properties for a object returned from an LSP server. So for example completionItem/resolve which resolves additional properties for a completion item previously returned from a textDocument/completion request.

Reply:
Will do, I was going between textDocument/moniker, textDocument/symbolMoniker, textDocument/resolveMoniker